### ISSUE TITLE
restrict -x usage to d > 1

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/swarm#153]
The option `--disable-sse3` (or `-x`) disables SSE3 and any other x86 extensions beyond SSE2 (which is always present on x86_64). SSE3 is only used when `d > 1`.

so, should swarm output an error or a warning if `-x` is used with `d < 2`?

This is what swarm does when the fastidious option is used out of context:
```
Error: Fastidious mode (specified with -f or --fastidious) only works 
when the resolution (specified with -d or --differences) is 1.
```
Silently accepting options that make no sense is ok, but in the long run I think it would benefit users if swarm requires "clean option sets". At the very least, users will have a clearer understanding of each option's role. We could have something like that: 
```
Error: --disable-sse3 or -x has no effect when d = 1 or d = 0 
(SSE3 instructions are only used when d > 1).
```

**Comment by frederic-mahe** [comment]
Test could be:
```sh
printf ">s1_1\nA\n>s2_1\nT\n"   | swarm -d 1 -x  # FAIL
printf ">s1_1\nAA\n>s2_1\nTT\n" | swarm -d 2 -x  # PASS
```

**Comment by torognes** [comment]
Added now in commit a1884502e2f1356989c287d20127927f16358f39.

**Comment by frederic-mahe** [comment]
fixed with commit a1884502e2f1356989c287d20127927f16358f39
